PROJECT_TECH_INFRA.md

```diff
@@ -0,0 +1,29 @@
+# PyLadies Project: Tech + Infra
+
+The proposal for this project can be found [here](https://github.com/pyladies/global-organizing/issues/35).
+
+### Mission
+
+Administer and maintain PyLadies tech and infrastructure. As such, Project lead(s) should be one of the PyLadies Admin account, however team members can be any self-identifying PyLadies member.
+
+### Responsibilities
+
+1. Manage `pyladies.com` (e.g. review pull requests for updating website on GitHub)
+
+### Team
+
+Name | GitHub Handle | PyLadies Chapter | Team Role |
+| --| --| --| --|
+| Mariatta | @Mariatta  | PyLadies Vancouver | Co-Lead |
+| Bethany G | @BethanyG   | PyLadies Oakland | Co-Lead |
+| Tania Allard | @trallard  | PyLadies Northwestern UK | Co-Lead |
+| Lorena Mesa* | @lorenanicole | PyLadies Chicago | Member |
+| Carol Willing | @willingc   | PyLadies Core | Member |
+| Jackie Kazil | @jackiekazil   | PyLadies DC | Member |
+| Flávia Cardoso | @fcardoso0917  | PyLadies Uruguay | Member |
+Amadikwa Joy N | @treasurechristain | Pyladies IMO | Member |
```

Comment:
missing | at start
```suggestion
| Amadikwa Joy N | @treasurechristain | Pyladies IMO | Member |
```

Reply:
Can we also ask for their Slack handle? I will add all team members into a Slack group.

Reply:
Sure why not